SU-SWS/saa_victoria_subtheme
# ADAPT-000: Fixed bad color definition that was not necessary.

# READY FOR REVIEW

# Summary
- there is a bad color definition - we should pull from decanter and this color is a mistake. removed it.

# Review By (Date)
- 7/17

# Urgency
- very noticeable for tuesday

# Steps to Test

1. pull down branch, clear cache
1. check a top banner with text and image, see that the background color is 'bright red' not vivid red.


# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket - n/a small fix
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
